prime-router/docs/getting-started/README.md

```diff
@@ -2,37 +2,172 @@
+docker ps
+# CONTAINER ID   IMAGE         COMMAND                  CREATED          STATUS          PORTS                    NAMES
+# 2962fb214203   postgres:11   "docker-entrypoint.s…"   57 minutes ago   Up 57 minutes   0.0.0.0:5432->5432/tcp   prime-router_postgresql_1
+```
+
+#### Run support services
+
+ReportStream depends on set of services to be up before running the main Azure service. The `cleanslate.sh` script
+starts a Postgres database but skips starting a few more that are otherwise started by default when `cleanslate.sh`
+start is run on a non-Apple processor:
+
+- Azurite - a simulator of Azure storage
+- Vault - a secret store
+- SFTP - an SFTP server
+- soap-webservice - SOAP web service emulator
+
+```bash
+docker-compose -f docker-compose.build.yml up --detach
+```
+
+Additionally, to ensure that Vault is running and the credentials are stored correctly, run the following (which is
+normally covered by `cleanslate.sh` on non-Apple processors):
+
+```bash
+docker-compose up --detach vault 1>/dev/null 2>/dev/null
+```
+
+You can take down these services by running `./gradlew composeDown` or `docker-compose down` command.
+For now, leave these services running and open up a new terminal session.
+</details>
+
+### Seed the Postgres db and vault
+
+To run tests, the Postgres DB and the credential vault need to be seeded with values.
+We will need to have ReportStream running for these steps to work (see previous steps).
+Again, we will use a Gradle task to do these steps.
+
+```bash
+./gradlew primeCLI --args "create-credential --type=UserPass --persist=DEFAULT-SFTP --user foo --pass pass"
+./gradlew reloadTables
+./gradlew reloadSettings
+```
+
+## Running ReportStream backend
+
+<details>
+  <summary>Docker Desktop</summary>
+If you are using Docker Desktop, verify that it is running prior to building or running ReportStream locally.
+</details>
+
+Building and running ReportStream requires a locally accessible PostgreSQL database instance that is initially setup and run by the `cleanslate.sh` script. This database instance runs as a Docker container defined by the `docker-compose.build.yml` file. You will need to start this database instance upon a workstation reboot by using the following command:
+
+```bash
+cd ./prime-router
+docker-compose --file "docker-compose.build.yml" up --detach
+```
+
+<details>
+  <summary>Gradle/Apple Silicon</summary>
+Use Gradle to launch ReportStream, as it will set up the environment variables that ReportStream needs.
+
+```bash
+./gradlew run
+```
+
+_Note:_ for quicker development you can use `./gradlew quickrun` which skips some long running tasks, but use with
+caution as it will not build the FatJar, run database related tasks, or run the tests.
+
+ReportStream should continue to run after launching. A successful build will appear like so:
+```
+<===========--> 90% EXECUTING [46s]
+> :azureFunctionsRun
+```
+A `ctrl-c` will kill the running ReportStream instance.
```

Comment:
More of an FYI, but sadly there is a bug in the azure functions that means that `ctrl-c` won't actually kill the process.  There is a special handler in build.gradle that kills process on restart

Reply:
Good to know. I changed the wording to "ctrl-c will escape the .. instance"